apps/avatax/next.config.js

```diff
@@ -56,7 +56,6 @@ const nextConfig = {
       "@saleor/app-sdk",
       "@trpc/server",
       "@trpc/client",
-      "@trpc/react-query",
```

Comment:
why removed?

Reply:
It turns out we need that for trpc to work properly - I bringed it back at [`9eb4f57` (#1790)](https://github.com/saleor/apps/pull/1790/commits/9eb4f579d07938c4ac0629ba9b0f8478445294df)